datasets/1.0/huggingface-mnist/metadata.json

```diff
@@ -2,6 +2,7 @@
   "@context": {
     "@language": "en",
     "@vocab": "https://schema.org/",
+    "arrayShape": "cr:arrayShape",
```

Comment:
Should all these modifications appear in a new `datasets/1.1` folder?

Reply:
(Could be done in another PR)

Reply:
As discussed offline, added a 1.1 folder with the new dataset. The 1.0 datasets are not cmodified. Will update the old datasets and put the new versions in 1.1 once repeated is deprecated.